### DISCUSSION TITLE
How to handle python translated to front-end

### DISCUSSION THREAD

**Discussion by jackiekazil** [mesa/mesa#1276] [Ideas]
I recently came across Gradio -- and I thought I was interesting how they were handling the front-end... 
https://www.gradio.app/

It is made by the Hugging Face people, I believe.

**Comment by rht** [comment]
That's neat. I read that it can run in a Colab notebook, which means it can run on a Jupyter notebook.

> If you're working out of colab notebook, a share link is always automatically created. It usually looks something like this: XXXXX.gradio.app. Although the link is served through a gradio link, we are only a proxy for your local server, and do not store any data sent through the interfaces.
